README.md

```diff
@@ -43,11 +46,18 @@ The project currently provides the following templates.
 
 ### Code Snippets
 
-| Snippet         | Shortcut     | Language      |
-|-----------------|--------------|---------------|
-| Test Fixture    | ntestfixture | C#            |
-| Test Method     | ntest        | C#            |
-| Test Case       | ntestcase    | C#            |
+| Snippet         | Shortcut     | Language      | Comment |
+|-----------------|--------------|---------------|---------|
+| Test Fixture    | ntestfixture | C#            | |
+| Test Method     | ntest        | C#            | |
+| Test Case       | ntestcase    | C#            | |
+| Test Case Source | ntestcasesource1    | C#  | Form 1 |
```

Comment:
Could the method-based one be `ntestcasesource` without the `1` since that's the most common?

Reply:
Yes, but should then the next start with 2 or 1 ?

Reply:
I'd say 2.